### PR TITLE
SecurePlayerAutomationHook

### PR DESCRIPTION
**Pull request by bindumadhav**

### CODE REVIEW

File: SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerEHLSPlayerActivity.java [hunk 1]
```diff
@@ -102,6 +107,11 @@ public void update(Observable arg0, Object argN) {
       return;
     }
 
+    // Automation Hook: to write Notifications to a temporary file on the device/emulator
+    String text="Notification Received: " + arg1 + " - state: " + player.getState();
+    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
+    playbacklog.writeToSdcardLog(text);
+
     Log.d(TAG, "Notification Received: " + arg1 + " - state: " + player.getState());
```
Comment (brol1dev) [inline SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerEHLSPlayerActivity.java]: Use the `text` variable in the `Log`. Let's not use two different with the same stuff. 

Please do that for all the Activities that applied the same change :)

File: SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerOPTPlayerActivity.java [hunk 2]
```diff
@@ -129,6 +134,11 @@ public void update(Observable arg0, Object argN) {
       return;
     }
 
+    // Automation Hook: to write Notifications to a temporary file on the device/emulator
+    String text="Notification Received: " + arg1 + " - state: " + player.getState();
+    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
+    playbacklog.writeToSdcardLog(text);
+
     Log.d(TAG, "Notification Received: " + arg1 + " - state: " + player.getState());
```
Comment (brol1dev) [inline SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerOPTPlayerActivity.java]: Same here about text variable

File: SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerOptionsPlayerActivity.java [hunk 3]
```diff
@@ -143,6 +148,11 @@ public void update(Observable arg0, Object argN) {
       return;
     }
 
+    // Automation Hook: to write Notifications to a temporary file on the device/emulator
+    String text="Notification Received: " + arg1 + " - state: " + player.getState();
+    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
+    playbacklog.writeToSdcardLog(text);
+
     Log.d(TAG, "Notification Received: " + arg1 + " - state: " + player.getState());
```
Comment (brol1dev) [inline SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerOptionsPlayerActivity.java]: Same here about text variable

File: SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerPlayerActivity.java [hunk 4]
```diff
@@ -109,6 +114,11 @@ public void update(Observable arg0, Object argN) {
       return;
     }
 
+    // Automation Hook: to write Notifications to a temporary file on the device/emulator
+    String text="Notification Received: " + arg1 + " - state: " + player.getState();
+    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
+    playbacklog.writeToSdcardLog(text);
+
     Log.d(TAG, "Notification Received: " + arg1 + " - state: " + player.getState());
```
Comment (brol1dev) [inline SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerPlayerActivity.java]: Same here about text variable

File: SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerPrePersonalizedPlayerActivity.java [hunk 5]
```diff
@@ -120,6 +124,11 @@ public void update(Observable arg0, Object argN) {
       return;
     }
 
+    // Automation Hook: to write Notifications to a temporary file on the device/emulator
+    String text="Notification Received: " + arg1 + " - state: " + player.getState();
+    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
+    playbacklog.writeToSdcardLog(text);
+
     Log.d(TAG, "Notification Received: " + arg1 + " - state: " + player.getState());
```
Comment (brol1dev) [inline SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerPrePersonalizedPlayerActivity.java]: Same here about text variable

**Review by brol1dev — APPROVED** [review]
Added the same comment I made in PR #153

**Comment by bindumadhav** [comment]
Made the changes based on CR comments

**Pull request merged**